### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#8539] [Help]
Hello everyone.

![Capture](https://user-images.githubusercontent.com/106697208/200169134-6bcde94c-aa15-4388-968f-1f8c774d3ada.PNG)

In this automation, I want to compare the status value after 120000 milliseconds and not after the row was created.
Explanation of what I want to do:
After the creation of the new row in the **_PF 15_** table, if the user will modify the row after 15 min from its creation (this means that the **_status_** will become = **_finished_**) then I will do nothing, Otherwise (after 15 min the user didn't modify the row , so that means that the **_status_** still equals **_waiting_**) so I'm going to modify it automatically using automation.

@melohagan 
Any idea ?

**Comment by melohagan** [comment]
Hey @rafikApp 

I think I understand the problem. The reason the delay doesn't work at the moment is because the **Row Created** trigger gives you access to the data at the time the row was created, not whatever the row data happens to be 120 seconds later after your delay step.

Ultimately to accomplish what you want, I recommend using the **Cron trigger** set to every 15mins (or a bit less if needed) and then do your conditional checks.

You can use the **Query rows** step with a filter, you should be able to get the ID of the row you want to update.

I believe there are examples of this in other discussions, if you search with the label *automation*

**Reply by rafikApp** [reply]
Hi @melohagan 
Thank you for your help. But I think there are two problems.

- the first is how to get the id of the created row
- the second how we will test 15 min exactly after the row creation
    for example:
              t1=0 : creation of a new row
              t2=5 : creation of the second row
              t3=15 : cron trigger (ok for the first row, but not ok for the second because of **t3-t2=10<15**)
              t4=20 : we should test the second line, but we don't have the **cron trigger** at this moment

**Reply by rafikApp** [reply]
@melohagan any suggetion ?
